internal/core/src/mmap/Column.h

```diff
@@ -153,11 +159,18 @@ class ColumnBase {
         column.size_ = 0;
     }
 
+    // Data() points at an addr that contains the elements
     virtual const char*
     Data() const {
         return data_;
     }
 
+    // MmappedData() returns the mmaped address
+    const char*
+    MmappedData() const {
```

Comment:
any difference compared to the Data()?

Reply:
`Data()` points at an array of elements, while `MmappedData()` points at the entire mmap-ed memory. They are the same for dense vectors but not the same for sparse vectors. `SparseFloatColumn` overrides `Data()` to return something different.